script/server/db/mysql.sql

```diff
@@ -31,10 +31,11 @@ CREATE TABLE IF NOT EXISTS `branch_table`
     `status`            TINYINT,
     `client_id`         VARCHAR(64),
     `application_data`  VARCHAR(2000),
-    `gmt_create`        DATETIME,
-    `gmt_modified`      DATETIME,
+    `gmt_create`        DATETIME(6),
```

Comment:
Milliseconds is enough.
User upgrades may forget this, might be better to add a long type timestamp column.

Reply:
I think the long type is less readable.